controllers/auth.go

```diff
@@ -55,6 +55,12 @@ func tokenToResponse(token *object.Token) *Response {
 func (c *ApiController) HandleLoggedIn(application *object.Application, user *object.User, form *form.AuthForm) (resp *Response) {
 	userId := user.GetId()
 
+	entryIpCheckError := object.CheckEntryIpByUser(user, c.Ctx.Request.RemoteAddr, c.GetAcceptLanguage())
```

Comment:
how about signup interface? we have fetch application and organization in this function many times, we should reduce unnecessary database access,